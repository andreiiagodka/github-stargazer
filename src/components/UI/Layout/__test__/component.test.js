import React, { Fragment } from 'react'
import { shallow } from 'enzyme'

import LayoutComponent from '..'

describe('<LayoutComponent />', () => {
  it('renders correctly', () => {
    const props = {
      header: 'Header',
      body: 'Body'
    }
    const wrapper = shallow(<LayoutComponent {...props} />)

    expect(wrapper).toMatchSnapshot()
  })
})
