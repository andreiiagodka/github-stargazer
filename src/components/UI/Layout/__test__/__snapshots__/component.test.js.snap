// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LayoutComponent /> renders correctly 1`] = `
<Fragment>
  <CardHeader>
    Header
  </CardHeader>
  <CardBody>
    Body
  </CardBody>
</Fragment>
`;
